pkg/metrics/queue.go

```diff
@@ -72,70 +92,89 @@ func InitQueueMetrics(name string) CoreQueueMetrics {
 }
 
 func (m *QueueMetrics) Reset() {
-	m.appMetrics.Reset()
-	m.ResourceMetrics.Reset()
+	m.appMetricsLabel.Reset()
+	m.appMetricsSubsystem.Reset()
+	m.ResourceMetricsLabel.Reset()
+	m.ResourceMetricsSubsystem.Reset()
 }
 
 func (m *QueueMetrics) IncQueueApplicationsRunning() {
-	m.appMetrics.With(prometheus.Labels{"state": "running"}).Inc()
+	m.appMetricsLabel.With(prometheus.Labels{"state": "running"}).Inc()
+	m.appMetricsSubsystem.With(prometheus.Labels{"state": "running"}).Inc()
```

Comment:
As far as I understand, `appMetricsSubsystem` will be removed and now it's kept for backward compatibility. If that's the case, please mark them with `"// Deprecated"` in the struct, maybe add a short comment as well.